frontend/public/components/dashboard/alerts-card/alert-item.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { Icon } from 'patternfly-react';
+
+import { getAlertSeverity, getAlertMessage, getAlertDescription } from './';
+
+const getSeverityIcon = severity => {
+  switch (severity) {
+    case 'critical':
+      return (
+        <Icon type="fa" name="exclamation-circle" className="co-alert__icon co-alert__icon--critical" />
+      );
+    case 'warning':
+    default:
+      return (
+        <Icon type="fa" name="exclamation-triangle" className="co-alert__icon co-alert__icon--warning" />
+      );
+  }
+};
+
+export const AlertItem: React.FC<AlertItemProps> = ({ alert }) => {
+  return (
+    <div className="co-alert__item">
+      {getSeverityIcon(getAlertSeverity(alert))}
+      <div className="co-alert__item-message">{getAlertDescription(alert) || getAlertMessage(alert)}</div>
+    </div>
+  );
+};
+
+type AlertItemProps = {
+  alert: object;
```

Comment:
this should have a better type. Can you reuse `Alert` from `monitoring.tsx` ?

Reply:
The `Alert` type from `monitoring.tsx` worked perfectly. Thanks!